### DISCUSSION TITLE
Adjusting width of posts

### DISCUSSION THREAD

**Discussion by chrisxfire** [hugo-sid/hugo-blog-awesome#46] [Q&A] [answered]
Hi, what's the *correct* way to adjust the width of texts in posts with this theme?

**Comment by hugo-sid** [comment, marked as answer]
Hi @chrisxfire. 

Welcome to Hugo blog awesome discussions.

## Solution

You have to change the value of  `$narrow-size` and `$spacing-full` variables, which are defined in `assets/sass/main.scss` file.

## Reasoning

The whole blog post is wrapped in a `<div>` with `wrapper` class. In order to change the width of text for individual posts, `max-width` property of this div should be changed.

https://github.com/hugo-sid/hugo-blog-awesome/blob/04afe81c635bf6129c841d9c9b72b13065e4eec0/assets/sass/_base.scss#L232-L234


As seen above, the value of `max-width` property is decided by the two variables `$narrow-size` and `$spacing-full`.  These variables are defined in `assets/sass/main.scss` (shown below).
https://github.com/hugo-sid/hugo-blog-awesome/blob/04afe81c635bf6129c841d9c9b72b13065e4eec0/assets/sass/main.scss#L53-L57

Ultimately, you should change the value of these two variables in order to change the width of text for individual posts.

## Conclusion
Let me know if this solution worked for you. In case it did, you can mark this as the answer. Otherwise, feel free to post your observations here. Don't forget to star the repo, if you liked this theme.

**Reply by chrisxfire** [reply]
Thanks for the detailed explanation — I learned something new from this!

**Reply by caprinux** [reply]
there seems to be more to it than that :(

the header and code block dont scale accordingly  (i.e. despite scaling up the width, the border of the code block is scaled up but the text within the code block still starts according to the original/default width) not sure why that is the case

**Reply by caprinux** [reply]
![image](https://github.com/hugo-sid/hugo-blog-awesome/assets/76640319/71cd73fa-eb1b-4d2c-8927-fd52031b70d9)

the offending line of code appears to be in `_code.scss`, where `max-width` is set to be `50rem`. by incrementing it to a larger number like `100`, the lines of code will correctly align with the code block once again.

**Comment by chrisxfire** [comment]
Changing `display` to `flexbox` (without changing `max-width`) also seems to resolve this.
